<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robots</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seleccione un robot:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboRobots"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnMostrar">
        <property name="text">
         <string>Mostrar Robot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelInfo">
      <property name="text">
       <string>Información del robot aquí...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetGrafica" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
